string name

---

bool valid					# true if input name is a registered person or voice commander, otherwise it is false
string message
mrta_vc/Agent person		# when voice commander is valid



